### PR TITLE
Airgap (#41)

### PR DESCRIPTION
**Pull request by kiransynopsys**
* airgap support.

### CODE REVIEW

**Review by spurohitsynopsys — APPROVED** [review]

File: synopsys-task/dist/index.js [hunk 1]
```diff
@@ -407,7 +406,7 @@ class SynopsysBridge {
                         }
                     }
                     else {
```
Comment (kishorikumar) [inline synopsys-task/dist/index.js:408]: Any specific reason, we are having 2 separate conditions for defaultPaht and SYNOPSYS_BRIDGE_INSTALL_DIRECTORY_KEY with almost same kind of operations? Can we try clubbing both in a single condition?

**Review by kishorikumar — CHANGES_REQUESTED** [review]

File: synopsys-task/dist/index.js (hunk 1 above)
Comment (kiransynopsys) [reply synopsys-task/dist/index.js:408]: Since we are having two different error message so we are using 2 separate condition.

Comment (kiransynopsys) [reply synopsys-task/dist/index.js]: restructured code.

File: synopsys-task/src/synopsys-task/synopsys-bridge.ts
[the remark this answers is not in the document]
Comment (kiransynopsys) [reply synopsys-task/src/synopsys-task/synopsys-bridge.ts:1]: restructured  code.

**Review by lsynopsys — APPROVED** [review]

**Pull request merged**